packages/block-editor/src/components/block-edit/edit.js

```diff
@@ -71,12 +72,23 @@ const EditWithGeneratedProps = ( props ) => {
 		return <EditWithFilters { ...props } context={ context } />;
 	}
 
-	// Generate a class name for the block's editable form.
-	const generatedClassName = hasBlockSupport( blockType, 'className', true )
-		? getBlockDefaultClassName( name )
-		: null;
+	const generatedClassNames = [];
```

Comment:
Alternatively, we could use a string that we append to 🤷‍♂️ 

```suggestion
	let generatedClassName = '';
```